AutomatticTracks/build.gradle

```diff
@@ -38,16 +38,24 @@ android {
     }
 }
 
-
-publish {
-    userOrg = 'automattic'
-    groupId = 'com.automattic'
-    uploadName = 'tracks'
-    artifactId = 'tracks'
-    desc = 'Android client for Nosara tracks (event tracking and analytics)'
-    publishVersion = android.defaultConfig.versionName
-    licences = ['MIT']
-    website = 'https://github.com/Automattic/Automattic-Tracks-Android'
-    dryRun = 'false'
-    autoPublish = 'true'
+afterEvaluate {
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                artifactId 'Automattic-Tracks-Android'
+                artifact("$buildDir/outputs/aar/AutomatticTracks-debug.aar")
+                description 'Android client for Nosara tracks (event tracking and analytics)'
+                groupId 'com.github.Automattic'
+                pom {
+                    licenses {
+                        license {
+                            name = 'GNU General Public License v2.0'
```

Comment:
This license change aligns to the one defined in project's initial commit https://github.com/Automattic/Automattic-Tracks-Android/commit/5652b539769f6874d81f112fe742a1ec7ad87b6e